.travis/spec_runner.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
@LocoDelAssembly Is this code based on existing code somewhere "out there"?  Please reference in comments if so.  If not- very nice.

Reply:
@mjy I believe that other searching for the correct bash syntax it was all mine.